sepolicy/mflinger.te

```diff
@@ -18,3 +18,6 @@ allow mflinger surfaceflinger_service:service_manager find;
 
 # allow mflinger to read the ion buffer used by mclient
 allow mflinger ion_device:chr_file r_file_perms;
+allow mflinger gpu_device:chr_file { read write };
+allow mflinger vendor_default_prop:file { getattr open read };
+allow mflinger vendor_display_prop:file { getattr open read };
```

Comment:
vendro_default_prop and vendor_display_prop are not defined by sailfish. I will check whether they are needed, or I will move those properties to sargo/bonito device's sepolicy directory.

Reply:
OK sounds good, thanks for checking.

Reply:
They are not needed. I removed it from new commit.